renderer.php

```diff
@@ -183,6 +183,27 @@ public function render_delete_acl_group_assignment_icon($courseid, $videoidentif
         return \html_writer::link($url, $icon);
     }
 
+    /**
+     * Render the link to change the visibility of a video.
+     *
+     * @param string $videoidentifier
+     */
+    public function render_change_visibility_icon($courseid, $videoidentifier, $visible) {
+        global $USER;
+        $url = new \moodle_url('/blocks/opencast/changevisibility.php',
+            array('identifier' => $videoidentifier, 'courseid' => $courseid, 'visible' => $visible, 'sesskey' => $USER->sesskey));
+        $text = get_string('changevisibility', 'block_opencast');
+
+        if($visible) {
```

Comment:
The icon should represent the current status and not the action to be done. Compare with hide/show for course modules.